src/core/acl.js

```diff
@@ -6,47 +6,59 @@ import { isDefined } from './utils';
 
 /**
  * The Acl class is used as a wrapper for reading and setting permissions on an entity level.
- *
- * @example
- * var entity = { _acl: {} };
- * var acl = new Kinvey.Acl(entity);
  */
 export class Acl {
   constructor(entity) {
     if (isPlainObject(entity) === false) {
       throw new KinveyError('entity argument must be an object');
     }
 
+    entity._acl = entity._acl || {};
```

Comment:
Even though this is a tiny (and harmless) change, generally it's bad form to make unrelated code changes in a docs PR.

Reply:
I agree. This was not meant to be in this PR. I will remove it.